Source/UI/Models/Badges/ChampionBadgeViewModel.cs

```diff
@@ -7,6 +7,7 @@ public class ChampionBadgeViewModel : IBadgeBaseViewModel
         internal const string FONT_AWESOME_CSS_CLASS = "fa-trophy";
         internal const string POPOVER_TEXT = "The Champion is the Player within the Gaming Group with the most wins of this game.";
 
+        public string GameName { get; set; }
```

Comment:
May not need this per previous comments. If we are only showing total number of championed games then this can go away.